data/json/items/gun/223.json

```diff
@@ -45,6 +45,29 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "famas",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "FAMAS G2" },
+    "description": "An assault rifle that was until recently used by French armed forces.  Famous for its high rate of fire, it was never available in a civilian version.",
+    "weight": "3800 g",
+    "volume": "4674 ml",
+    "price": 540000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": "223",
+    "ranged_damage": -2,
+    "dispersion": 180,
+    "durability": 8,
+    "min_cycle_recoil": 1350,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
```

Comment:
The G2 version uses the standard mags. The ones that were imported had a proprietary 25 rounder.

Reply:
Added the magazine and changed that as appropriate.